### ISSUE TITLE
new empty HMM segfaults when saved to file

### ISSUE THREAD

**Issue by RuneBlaze** [althonos/pyhmmer#5]
```python
import pyhmmer
from pyhmmer.plan7 import HMMFile, HMM
from pyhmmer.easel import Alphabet
print(pyhmmer.__version__)
newhmm = HMM(10, Alphabet.dna())
with open("newhmm.hmm", "w+b") as fh:
  newhmm.write(fh)
```

Output:
```
0.3.0
Segmentation fault (core dumped)
```

I don't know if this is a bug or not, but this seems a bit counterintuitive at least. If this is a bug, is there any chance that I can know why this is segfaulting?

**Comment by althonos** [comment]
Hi @RuneBlaze, I was able to track it down in the HMMER source, and it is a bug with their writer (https://github.com/EddyRivasLab/hmmer/blob/master/src/p7_hmmfile.c#L649...L654). Basically, it reads from the HMM consensus line before checking the flags to see if the HMM has one; for newly created HMMs (without consensus), this means reading from a NULL pointer, hence the SEGFAULT.

I'll open a PR to fix it in their code and update here when it gets merged.

**Comment by RuneBlaze** [comment]
Thanks for taking the time to tracking down and fixing it in the original HMMER source!

**Comment by althonos** [comment]
Officially fixed in `v0.10.0` since it updates to HMMER `v3.4`!
